tf-modules/cumulus-rds-tf/db-provision-lambda.tf

```diff
@@ -6,14 +6,14 @@ resource "random_string" "db_pass" {
 
 module "provision_database" {
   count = var.provision_user_database ? 1 : 0
-  source                      = "./db-provision-user-database"
-  prefix                      = var.prefix
-  subnet_ids                  = var.subnets
-  rds_security_group          = aws_security_group.rds_cluster_access.id
-  rds_admin_access_secret_arn = aws_secretsmanager_secret_version.rds_login.arn
-  tags                        = var.tags
-  permissions_boundary_arn    = var.permissions_boundary_arn
-  vpc_id                      = var.vpc_id
-  rds_user_password           = var.rds_user_password == "" ? random_string.db_pass.result : var.rds_user_password
-  rds_connection_heartbeat    = true
+  permissions_boundary_arn               = var.permissions_boundary_arn
+  prefix                                 = var.prefix
+  rds_admin_access_secret_arn            = aws_secretsmanager_secret_version.rds_login.arn
+  rds_connection_timing_configuration    = var.rds_connection_timing_configuration
+  rds_security_group                     = aws_security_group.rds_cluster_access.id
+  rds_user_password                      = var.rds_user_password == "" ? random_string.db_pass.result : var.rds_user_password
+  source                                 = "./db-provision-user-database"
+  subnet_ids                             = var.subnets
+  tags                                   = var.tags
+  vpc_id                                 = var.vpc_id
```

Comment:
Are we intentionally looking to move toward alpha-sorting module params?

Reply:
I've been trying to alpha sort env vars/*.js/*.ts params, but I think there's room for module parameters to be layed out in a logically consistent paragraph structure.   That said, in absence of one alpha sort works.